routes/api.js

```diff
@@ -142,11 +149,13 @@ function map2dorita (source, method, hasArgs) {
     if (hasArgs) {
       if (!req.body) return next('Invalid arguments.');
     }
+    var hasBody = req.body !== undefined
```

Comment:
this is not required because hasArgs is true for cleanRoom action
(also there is white line here)